src/core/Microsoft.PowerToys.Settings.UI/Views/GeneralPage.xaml

```diff
@@ -83,7 +83,7 @@
                           IsEnabled="{Binding Mode=TwoWay, Path=IsElevated, Source={StaticResource eventViewModel}}"
                           IsOn="{Binding Mode=TwoWay, Path=RunElevated, Source={StaticResource eventViewModel}}"/>
 
-            <HyperlinkButton NavigateUri="https://github.com/microsoft/PowerToys/blob/master/doc/devdocs/run-as-admin-detection.md">
+            <HyperlinkButton NavigateUri="https://github.com/microsoft/PowerToys/wiki/PowerToys-and-running-as-Administrator">
```

Comment:
@crutkas 
Do we have an aka.ms link for that?

Reply:
Ditto for all other URIs in this file.

Reply:
this actually should be an aka link... https://aka.ms/powertoysDetectedElevatedHelp

Reply:
I just pushed out the change with the aka.ms link.

Reply:
I replied to issue, I injected back in a temp redirect page